### PR TITLE
MTM-53127 Added icon selector info to the docs.

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

File: content/cockpit/cockpit-configuration.md [hunk 1]
```diff
@@ -56,6 +56,6 @@ The initial home dashboard shows a number of pre-installed widgets. These widget
 
 Click **Reset dashboard** to undo any changes to your home dashboard. This reverts all changes to the dashboard and returns it to the initial state.
 
-#### To hide the navigator {#to-hide-the-navigator}
+#### Title, icon and navigator collapse {#to-select-icon-change-title-hide-navigator}
 
-Finally, you can specify if the navigator should be hidden on start up. By default, the navigator is displayed on start up.
+Under the **Title, icon and navigator collapse** section you can edit the title of the application, change the icon and finally, you can specify if the navigator should be hidden on start up. By default, the navigator is displayed on start up.
```
Comment (BeateRixen) [inline content/cockpit/cockpit-configuration.md:61]:
```suggestion
Under the **Title, icon and navigator collapse** section you can modify the application title and icon, and you can specify if the navigator should be hidden on start up. By default, the navigator is displayed on start up.
```

File: content/cockpit/cockpit-configuration.md [hunk 2]
```diff
@@ -56,6 +56,6 @@ The initial home dashboard shows a number of pre-installed widgets. These widget
 
 Click **Reset dashboard** to undo any changes to your home dashboard. This reverts all changes to the dashboard and returns it to the initial state.
 
-#### To hide the navigator {#to-hide-the-navigator}
+#### Title, icon and navigator collapse {#to-select-icon-change-title-hide-navigator}
```
Comment (BeateRixen) [inline content/cockpit/cockpit-configuration.md:59]:
```suggestion
#### Title, icon and navigator collapse {#title-icon-and-navigator-collapse}
```

File: content/standard-tenant/ecosystem-bundle/application-properties.md [hunk 3]
```diff
@@ -56,5 +56,15 @@ In the **Properties** tab, each application will show the following information,
 <td style="text-align:left">Automatically created</td>
 <td style="text-align:left">Specified by the user; for example, if you use "hello" as application path, the URL of the application will be "/apps/hello"</td>
 </tr>
+<tr>
+<td style="text-align:left">Select Icon</td>
```
Comment (BeateRixen) [inline content/standard-tenant/ecosystem-bundle/application-properties.md:60]:
```suggestion
<td style="text-align:left">Select icon</td>
```

File: content/standard-tenant/ecosystem-bundle/application-properties.md [hunk 4]
```diff
@@ -56,5 +56,15 @@ In the **Properties** tab, each application will show the following information,
...
+<td style="text-align:left">Opens a modal in which a wide variety of icons can be selected.</td>
```
Comment (BeateRixen) [inline content/standard-tenant/ecosystem-bundle/application-properties.md:61]:
```suggestion
<td style="text-align:left">Provides a variety of icons from which an icon for the application can be selected.</td>
```

File: content/standard-tenant/ecosystem-bundle/custom-applications.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ Duplicating an application might be useful if you want to customize a subscribed
 1. Click **Add application** at the top right of the **Applications** tab.
 2. In the upcoming dialog, select **Duplicate existing application**.
 3. Select the desired application from the dropdown list, for example "Cockpit".
-4. In the next window, provide a name for the application, an application key to identify the application, and a path as part of the URL to invoke the application. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.
+4. In the next window, provide a name for the application, an application key to identify the application, a path as part of the URL to invoke the application and finally select an icon for the new applicaiton via the select icon modal. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.
```
Comment (BeateRixen) [inline content/standard-tenant/ecosystem-bundle/custom-applications.md:75]:
```suggestion
4. In the next window, provide a name for the application, an application key to identify the application, and a path as part of the URL to invoke the application.  Finally select an icon for the new application from the available icons. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.
```

**Comment by BeateRixen** [comment]
@eniosultan Please have one more approval from your team.

**Review by DawidSAG — APPROVED** [review]

**Pull request merged**